.env.dist

```diff
@@ -1,5 +1,7 @@
 # Configuration
 RALPH_APP_DIR=/app/.ralph
+FORCE_COLOR=1
+TERM=xterm-256
```

Comment:
I personally don't mind with this env var approach. Maybe add a comment just to be clear for Rich.